Week2/homework/maartjes-work.js

```diff
@@ -46,16 +47,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const moreThanOrEqualTo2Hours = tasks.filter(element => element.duration >= 120);
+  const totalDuration = moreThanOrEqualTo2Hours.reduce(
+    (total, currentElement) => total + currentElement.duration,
+    0,
+  );
+  const exactToEarn = (totalDuration / 60) * hourlyRate;
+  return exactToEarn;
```

Comment:
Cool that you did the `filter` first as it prevents having to do work on elements which are not relevant. Nice!

Reply:
@nudge , 
Thanks for reviewing the homework.
Best regards